### ISSUE TITLE
Add an Example for DeepBeliefNetwork Class

### ISSUE THREAD

**Issue by functionsoft** [accord-net/framework#775]
Please add an example for DeepBeliefNetwork Class.

TODO (optional): Describe a specific scenario you would like to see addressed.

Help Topic: http://accord-framework.net/docs/html/T_Accord_Neuro_Networks_DeepBeliefNetwork.htm

**Comment by cesarsouza** [comment]
Hi @functionsoft,

Thanks a lot for opening the issue! Deep Belief Networks have long been superseeded by supervised, end-to-end learnt networks trained using ReLU, Dropout and BatchNorm. For unsupervised tasks, those have also been superseeded by deep auto-encoders using the same tricks. As such, I wouldn't recommend using DBN in new applications right now.

For the moment, I think it would not be a good idea to add examples to networks in the Accord.Neuro namespace because soon they will be replaced by networks from [Keras Sharp](https://github.com/cesarsouza/keras-sharp) (which can be seen as the next version of Accord.Neuro).

As such, unless you really and absolutely need to use deep belief networks in your application, I would recommend waiting until better models have been made available through the Keras Sharp project (please feel free to open issues and documentation requests there by the way!).

Regards,
Cesar
